### ISSUE TITLE
documetation for NODE and EDGE is missing

### ISSUE THREAD

**Issue by sameeraroshan** [almende/vis#914]
hi.. lot has been changed with visjs 4+ and i cannot find the documentation for node and edge. only nodes and edges configurations are there. This blocks me form updating the library i created for vaadin  https://vaadin.com/directory#!addon/visjs-addon to 4+. could you provide the necessary documentation.

**Comment by AlexDM0** [comment]
Hi,

As stated above the nodes and edges options, those are the same. 

Regards

**Comment by sameeraroshan** [comment]
i dint get your message. for example previous edge had

String from;
String to;
int value;
int stabilizationIterations = 1000;
String label;
String title;

etc

i couldn't find those information or alternatives in current
documentation under Edges. am i missing something here?

**Comment by AlexDM0** [comment]
I see to and from are indeed missing. Other then that, all options for edges are on the docs. 

Regards

**Comment by sameeraroshan** [comment]
so previously you had Edge and Edges / Node and Nodes . is it changed to
only Edges without EDGE and Nodes without NODE?

**Comment by AlexDM0** [comment]
Hi,

No. I think you are mixing up a few things. 

In the global options object (which you give to the network) we have nodes and edges as properties that can be filled with the options we have in the docs. This is exactly the same as before except that certain options have been renamed, moved or removed. 

The individual nodes and edges you supply as data also have options. These options are exactly the same as the ones you can give to the global object. To and from are indeed missing from the list at the moment. Ill fix that. 

We expect our users do not supply the id, label and other properties that are by definition bound to individual items to the global object. 

To sum up, the idea is exactly the same as before but since the items are the same, we do not list them separately. Please read the introduction texts on the nodes and edges pages for more information.  

Regards,

Alex

**Comment by sameeraroshan** [comment]
ok. now i got it. thank you alex.

**Comment by sameeraroshan** [comment]
Hello there,

I just signed the petition "BBC: The terrorist bomb explosions in Sri Lanka
and misleading reporting of BBC" and wanted to see if you could help by
adding your name.

Our goal is to reach 25,000 signatures and we need more support. You can
read more and sign the petition here:

http://chng.it/Mq6qwyxcQT

Thanks!
sameera
